SignalServiceKit.podspec

```diff
@@ -17,15 +17,37 @@ An Objective-C library for communicating with the Signal messaging service.
 
   s.homepage         = "https://github.com/WhisperSystems/SignalServiceKit"
   s.license          = 'GPLv3'
-  s.author           = { "Frederic Jacobs" => "github@fredericjacobs.com" }
+  s.author           = { "Whisper Systems" => "ios@whispersystems.com" }
   s.source           = { :git => "https://github.com/WhisperSystems/SignalServiceKit.git", :tag => s.version.to_s }
-  s.social_media_url = 'https://twitter.com/FredericJacobs'
+  s.social_media_url = 'https://twitter.com/WhipserSystems'
+
+  deprecation_message = <<EOS
+installing SignalServiceKit via the Signal-iOS repository.
+
+To get future updates, simply point your Podfile at the new location:
+
+    pod 'SignalServiceKit', git: 'https://github.com/WhisperSystems/SignalServiceKit.git'
+
+To this:
+
+    pod 'SignalServiceKit', git: 'https://github.com/WhisperSystems/Signal-iOS.git'
+
+Sorry for the disruption!
+
+EOS
+
+  s.deprecated_in_favor_of = deprecation_message
 
   s.platform     = :ios, '8.0'
   #s.ios.deployment_target = '8.0'
   #s.osx.deployment_target = '10.9'
   s.requires_arc = true
-  s.source_files = 'src/**/*.{h,m,mm}'
+
+  # By not including any actual files, upgrading users will see
+  # that they need to point upgrades to the new source at
+  # https://github.com/WhisperSystems/Signal-iOS
+  # Details in README.md
+  s.source_files = 'README.md'
```

Comment:
Note that this will effectively break HEAD of `github.com/WhisperSystems/SignalServiceKit`. Existing users running `pod install` will be fine, but they will hit it the next time they run `pod update` or potentially if they're not tracking a Podfile.lock (which they should be)